### DISCUSSION TITLE
[功能建议]: 电影和剧集重命名格式能够增加 一个显示帧率变量，比如60fps和120fps。

### DISCUSSION THREAD

**Discussion by xwndo** [jxxghp/MoviePilot#3487] [讨论]
### 当前程序版本

v1.8.8

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

MOVIE_RENAME_FORMAT：  电影和剧集重命名格式能够增加 一个显示帧率变量，比如60fps和120fps。

![截图_20240515145636](https://github.com/jxxghp/MoviePilot/assets/44762565/d3d353ee-9527-4166-85ac-8cb169e3532b)

**Comment by ArthurSmoke** [comment]
+1 现在国产剧资源中 60fps的资源越来越多了

**Comment by 13bit-X** [comment]
如果可以，能不能在搜索结果加一个帧率的标签和筛选。
